packages/create-commandkit/src/utils.ts

```diff
@@ -22,14 +26,22 @@ export const textColors = {
     ts: ['#2480c5', '#2480c5'],
 };
 
+const baseDependencies = ['commandkit', 'discord.js', 'dotenv'];
+
 export const dependencies = {
-    js: ['commandkit', 'discord.js', 'dotenv'],
+    js: {
+        dependencies: baseDependencies,
+    },
+    ts: {
+        dependencies: baseDependencies,
+        devDependencies: ['@types/node', 'typescript'],
+    },
 };
 
 export const commands = {
     init: {
         npm: 'npm init -y',
-        yarn: 'yarn init -y; yarn config set nodeLinker node-modules',
+        yarn: 'yarn init -y; yarn config set nodeLinker node-modules; yarn set version stable',
```

Comment:
```suggestion
        yarn: 'yarn init -2; yarn config set nodeLinker node-modules;',
```

Reply:
@twlite
- Using `yarn init -2` creates a bunch of useless files (some of them are created by create-commandkit), including a `.pnp.cjs` file
- Using `yarn init -y; yarn config set nodeLinker node-modules` will not create the extra files and will work as expected

<img width="328" alt="image" src="https://github.com/underctrl-io/commandkit/assets/140819896/9472781f-b0cf-4a1e-aa72-053c43eadbcd">